sources/BigInt.swift

```diff
@@ -259,6 +259,17 @@ extension BigInt: IntegerArithmetic {
     public static func %(a: BigInt, b: BigInt) -> BigInt {
         return BigInt(abs: a.abs % b.abs, negative: a.negative)
     }
+  
+    /// Return the result of `a` mod `b`.
+    public static func modulus(a: BigInt, b: BigInt) -> BigInt {
```

Comment:
Please use underscores to eliminate argument labels here; `a` and `b` are not meaningful.